.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,30 @@
 ---
 name: Bug Report
-about: Report a bug in beluga.
+about: Create a report to help us improve.
 title: ''
 labels: bug
 
 ---
 
-## Required info
-- Version or commit hash:
-  - <!-- Output of git rev-parse HEAD or release version  -->
-
-## Steps to reproduce issue
-<!--
-Detailed instructions on how to reliably reproduce this issue.
-Try to share a minimum reproducible example, see http://sscce.org/.
--->
-
-## Expected behavior
-<!--
-What are you expecting as a result of the above steps.
--->
-
-## Actual behavior
-<!--
-What is actually happening when the steps are followed.
-If your program crashes unexpectedly, hangs, etc, please use the debugger to share a traceback.
--->
-
-## Additional information
-<!--
-Here you can add:
-- Permalinks to a related code snippet https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet
-- The version or PR where the failure was introduced.
-- Other relevant info not in the above categories.
--->
+### Bug description
+_A clear and concise description of what the bug is._
+
+
+**Platform (please complete the following information):**
+ - OS: [e.g. Ubuntu Focal]
+ - `Beluga` version: [e.g. tag, commit sha]
```

Comment:
It needs not be a comment. 

We can for instance explicitly ask for the output to `lsb_release -a` and `git rev-parse`  instead of "OS" and "Beluga version". That also has the advantage of standardizing the information they provide.

In any case, feel free to leave as is now.

Reply:
I couldn't find a way to write it that I liked. I'll leave as is.